addons/social_share/models/url.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import base64
+import secrets
+from datetime import datetime, timedelta
+
+from odoo import api, fields, models
+
+
+class SocialShareUrl(models.Model):
+    _name = 'snshare.url'
+    _description = 'Social Share URL resolver'
```

Comment:
Sorry, at first glance I did not get what this model was for, could you add a small class comment to explain its purpose? :pray: